docs/mine-hnt/validators/troubleshooting.mdx

```diff
@@ -20,24 +20,56 @@ This can be run directly on the Docker container as well:
 
 ```docker exec validator tail -f /var/data/log/console.log```
 
-
 ## Compromised server
 If you find that the server has been compromised just wipe it (save the blocks to avoid having to resync the entire blockchain), close the hole, stand up a new validator and then transfer the stake to it.
 
-
 ## Syncing to the blockchain
 Syncing to the blockchain could be impacted for the following reasons:
 * slow internet connection
 * your validator is disconnecting
 * you just launched an AMI instance (if using AWS)
 
+## Grab the latest blessed snapshot from Helium
+* On a running Validator run `miner snapshot list`
+* Find the highest "Height" block and download the snapshot from Helium via `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`
+* Move that file into your `validator_data` directory
+* Temporarily pause transaction sync via ```miner repair sync_pause```
+* Cancel any in-progress transaction sync via ```miner repair sync_cancel```
+* Load snapshot from downloaded file ```miner snapshot load /var/data/snap-<height>```
```

Comment:
nit: This load cmd may "timeout" and look like it fails, but the load continues in the background. Same note applies to load from a trusted validator. Not sure if you need to mention this or not.